tests/data/example_test_nones.model.csv

```diff
@@ -6,7 +6,7 @@ Sex,,,"Female, Male, Other",,,TRUE,DataProperty,,
 Year of Birth,,,,,,FALSE,DataProperty,,
 Diagnosis,,,"Healthy, Cancer",,,TRUE,DataProperty,,
 Cancer,,,,"Cancer Type, Family History",,FALSE,ValidValue,,
-Cancer Type,,,"Breast, Colorectal, Lung, Prostate, Skin",,,TRUE,DataProperty,,
+Cancer Type,,,"Breast, Colorectal, Lung, Prostate, Skin, None",,,TRUE,DataProperty,,

```

Comment:
`Cancer Type` needs to be added to the `Depends On` column of the `MockComponent` component; without it, the jsonschema validation won't actually validate the `Cancer Type` column of the test manifest because it's not a part of the component and the valid values won't be verified. Alternatively, a `Patient` type manifest could also be used.

Reply:
I spent some time and added the suggestion, but then no matter what I add into the `Cancer Type` of the `MockCompontent` manifest nothing happens. Would I be expecting that `validate_manifest_rules` would be triggering any errors or warning if a value isn't one in the list that it could be?

I also tried modifying one of the CSV files in this test and I couldn't get anything to fail for any random data I add into the CSV: https://github.com/Sage-Bionetworks/schematic/blob/schematic-210-pandas-remove-none/tests/test_validation.py#L73-L81

Would you be willing to add a commit with the changes you're suggesting so I can learn what needs to be done?

Reply:
@BryanFauble sure, I opened #1555 to address